src/components/Header.jsx

```diff
@@ -0,0 +1,26 @@
+import { useContext } from "react";
+import Container from "react-bootstrap/Container";
+import Nav from "react-bootstrap/Nav";
+import Navbar from "react-bootstrap/Navbar";
+import { UserContext } from "../context/UserContext";
+
+function Header() {
+  const { user } = useContext(UserContext);
+  return (
+    <Navbar bg="light" expand="lg">
+      <Container>
+        <Navbar.Brand href="/">NorthCoders News</Navbar.Brand>
```

Comment:
I can see your using this react bootstrap component for you nav bar, something to be aware of is they can be awkward to dev lwith your links later on, although you've got articles working fine :)